scripts/release.py

```diff
@@ -204,6 +207,11 @@ class Release(object):
 
     # pylint: disable=R0913
     def __init__(self, repo, release_info):
+        """
+        :type repo: Repository
```

Comment:
Which syntax is that? Sphinx or PEP 287 docstrings? We're used to follow PEP 257 conventions.

Reply:
That's [PEP 484](https://www.python.org/dev/peps/pep-0484/), [PEP 257](https://www.python.org/dev/peps/pep-0257/) is only about docstrings conventions not about type hinting for IDEs.